docs/fx-integration.md

```diff
@@ -39,6 +42,26 @@ Firefox uses [Relay's REST API][relay-rest-api]. In particular, it uses these en
 - `GET /api/v1/profiles/` to get the user's Relay profile data.
 - `GET|POST /api/v1/relayaddresses/` to get and make relay addresses for the user.
 
+## Testing with local, dev, or stage Relay
+
+As of 2023-07-13, Relay integration is only enabled for 100% of Nightly users, so to
+test reliably you'll need to use Nightly. By 2023-08-29, it should be available for all
+Firefox users signed into the browser with their FXA.
```

Comment:
This seems better. Is there a "proper" abbreviation for Firefox Accounts? I've always used "FxA", but these changes are FXA.

Reply:
Changed to "FxA" everywhere.